pages/linux/journalctl.md

```diff
@@ -23,6 +19,10 @@
 
 `journalctl -u {{unit}}`
 
+- Show logs from last unit start:
+
+`journalctl _SYSTEMD_INVOCATION_ID=$(systemctl show --value --property=InvocationID {{unit}})`
```

Comment:
I'm not sure I understand `_SYSTEMD_INVOCATION_ID` here. Can you link me to some documentation?

Also, I see you're using a subcommand here. We try to document the command in question (in this case `journalctl`), rather than its interaction with other commands.

Can this command be rephrased to remove the subcommand call here?

In addition, the `$()` syntax may be shell-specific - e.g. it wouldn't work in [fish](https://fishshell.com/), for example.

Reply:
>  Can you link me to some documentation?

Fields description can be found in [man systemd.journal-fields](https://www.freedesktop.org/software/systemd/man/latest/systemd.journal-fields.html).

```
       _SYSTEMD_INVOCATION_ID=
           The invocation ID for the runtime cycle of the unit the message was generated in
```

> We try to document the command in question
> Can this command be rephrased to remove the subcommand call here?

There is no other way to get logs from the last unit start. Additionally, it is not clear how to do it after reading `man journalctl` (unlike existing examples) that is why I decided to make this PR.

> In addition, the $() syntax may be shell-specific - e.g. it wouldn't work in [fish](https://fishshell.com/), for example.

I don't use fish/other shells. If you can a suggest syntax that will work everywhere I will be happy to apply your suggestion.

Reply:
Thanks for this. Let's get a second opinion from another maintainer on this then.

> I don't use fish/other shells.

Right, but other people might.

Reply:
> Ok, should we use backticks?

Backticks don't work in fish and other non-POSIX compliant shells.

I don't think there's a way to write command substitution that will work across ***all*** shells. `$(command)` is the most used method, so I guess it's the most appropriate to use that here.

Reply:
Hmm okay, @acuteenvy. Makes sense.